src/libs/protocols/protos/confidential_data_hub.proto

```diff
@@ -46,4 +64,12 @@ message GetResourceResponse {
 
 service GetResourceService {
     rpc GetResource(GetResourceRequest) returns (GetResourceResponse) {};
+}
+
+// ImagePullService is used to pull images from a remote registry
+// The workdir to store the original image layers and configurations
+// are all kept by the CDH server, which the caller does not need to
+// know.
```

Comment:
As stated, the proto definition does not need to know about this ;)

Reply:
Thanks, deleted.